packages/contracts-bedrock/test/L2/SuperchainERC20.t.sol

```diff
@@ -115,4 +115,18 @@ contract SuperchainERC20Test is Test {
         assertEq(superchainERC20.totalSupply(), _totalSupplyBefore - _amount);
         assertEq(superchainERC20.balanceOf(_from), _fromBalanceBefore - _amount);
     }
+
+    /// @notice Tests that the `supportsInterface` function returns true for the `IERC7802` interface.
+    function test_supportInterface_succeeds() public view {
+        assertTrue(superchainERC20.supportsInterface(type(IERC165).interfaceId));
+        assertTrue(superchainERC20.supportsInterface(type(IERC7802).interfaceId));
```

Comment:
As per comment above, we might want to add in all the interfaces we support here.